rise/lib/resolvers/auth.ts

```diff
@@ -0,0 +1,54 @@
+import * as auth from '../services/auth'
+import { MutationResolvers } from '../__generated__/graphql'
+import { gql } from 'apollo-server-express'
+
+export const typeDefs = gql`
+  type AuthPayload {
+    id: ID!
+    token: String!
+    email: String!
+  }
+  type LogoutResponse {
+    status: String!
+    message: String!
+  }
+  input userInput {
+    email: String!
+    password: String!
+  }
+
+  extend type Mutation {
+    login(input: userInput): AuthPayload!
```

Comment:
Both of these calls might be better represented as a `Maybe<AuthPayload>`, or at least we'll have to make a decision on the behaviour of these mutations, I think. 

Currently they take a "maybe" input but always return an AuthPayload, perhaps this needs another round of thinking? 🤔